netezza-plugin/src/test/java/io/cdap/plugin/netezza/NetezzaPluginTestBase.java

```diff
@@ -51,7 +51,7 @@
 import java.util.Map;
 import java.util.TimeZone;
 
-public class NetezzaPluginTestBase extends DatabasePluginTestBase {
+public abstract class NetezzaPluginTestBase extends DatabasePluginTestBase {
```

Comment:
how come these are being changed to abstract classes?

Reply:
As *TestBase classes don’t have methods with @Test annotation so we received tests fail for each plugin with cause: “no runnable methods”.